### PR TITLE
[CAZB-2866] Add a lock on CAZ if one user is paying for it

### PR DESCRIPTION
**Pull request by likeuwill**
## Motivation and Context

https://eaflood.atlassian.net/browse/CAZB-2866
## Checklist:

- [ ] It contains only changes required by issue (does not contain other PR)
- [ ] Includes link to an issue (if apply)
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: app/controllers/concerns/calculating_chargeability.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+# Module used to check upload job status
+module CalculatingChargeability
```
Comment (tkaczu) [inline app/controllers/concerns/calculating_chargeability.rb:4]: I think module should be in a noun version - `ChargeabilityCalculator`

File: config/routes.rb [hunk 2]
```diff
@@ -8,10 +8,10 @@
   devise_scope(:user) { root to: 'sessions#new' }
 
   scope controller: 'logout' do
-    get 'sign-out', to: 'logout#sign_out_page'
```
Comment (tkaczu) [inline config/routes.rb:11]: Why this change was introduced?

**Review by tkaczu — APPROVED** [review]

File: config/routes.rb (hunk 2 above)
Comment (likeuwill) [reply config/routes.rb:11]: Because we have next path for sign `https://dev-fleet.paycleanairzonecharge.co.uk/users/sign_in`
plus all routes for application is underscored
